client/modules/IDE/reducers/files.js

```diff
@@ -18,10 +18,13 @@ const defaultHTML =
     <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.11/addons/p5.sound.min.js"></script>
     <link rel="stylesheet" type="text/css" href="style.css">
     <meta charset="utf-8" />
+
   </head>
   <body>
+    <section aria-label='accessible output' id='accessible-outputs'></section>
```

Comment:
rather than add the accessibility elements and css to the default html file, i think it would be better to inject them in the `PreviewFrame.jsx` file, in the function `injectLocalFiles`. As it is now, a user could accidentally delete these elements from the HTML and then the accessibility features would be broken.

Reply:
@catarak - I think some of the questions we had for you were deleted after we made the new commits :) 
@lm-n and myself were thinking it should be a part of the HTML (at the risk of being deleted) , the same way as p5-dom or p5-sound. This way if someone decides to share their code, they can share it with the accessibility lib and so the code stays accessible beyond the editor. What do you think about this?

Reply:
got it. should it be added by default or only if accessibility features are turned on? and also, you want the script to point to a specific release of the library? or a specific commit? or to the latest on master?

Reply:
I think we tried to add it only if accessibility features are added but there was a problem? @lm-n ?